sound/Music.java

```diff
@@ -123,8 +128,9 @@ private void establishSynthesizer(int numCh, int[] instr) {
      * Create a list of the available instruments.
      * @return list of instruments available in this object's synth
      */
-    public java.util.List<Instrument> AvailableInstruments() {
-        java.util.List<Instrument> instruments = new java.util.ArrayList<Instrument>();
+    public java.util.List<Instrument> availableInstruments() {
+        java.util.List<Instrument> instruments =
+                  new java.util.ArrayList<Instrument>();
```

Comment:
Java coding convention: break lines before an operator

The assignment operator should be on the following line.

See: _4.2 Wrapping Lines_ in https://www.oracle.com/technetwork/java/codeconventions-150003.pdf

Reply:
For all break line fixes, I broke at comma or before operator and then most if not all were indented 8 spaces for consistency and readability.

Reply:
That sounds right.  So this line needed to be broken before the assignment operator (rather than after).